src/it/project-004/pom.xml

```diff
@@ -49,7 +49,7 @@
       </plugin>
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.3</version>
+        <version>3.5.4</version>
```

Comment:
We can use a property here like `@version.maven-surefire@` (to check in parent)

Reply:
@slawekjaranowski is it possible to set this property globally for all tests?

Reading the [doc](https://maven.apache.org/plugins/maven-invoker-plugin/examples/invoker-properties.html), it seems I have to add this property in every single test project. Do I miss something?